rest_orm/models.py

```diff
@@ -11,34 +13,33 @@ class BaseModel(object):
     __metaclass__ = ModelRegistry
 
 
-class AdaptedModel(BaseModel):
+class Model(BaseModel):
     """A flat representation of a single remote endpoint."""
 
+    def loads(self, data):
+        """Load a JSON string to a flattened dictionary."""
+        return self.load(json.loads(data))
+
+    def load(self, data):
+        """Flatten a nested dictionary."""
+        response = {}
+        for field_name in dir(self):
+            field = getattr(self, field_name)
+            if not isinstance(field, Field):
+                continue
+            response[field_name] = copy(field.deserialize(data))
```

Comment:
@morgan This is the relevant part that was causing the issue.  Using `copy` prevents an accidental reference to the field class from being passed and updated.